tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -130,3 +160,11 @@ export const hasPermissionToShare = async (
   await Collaborator.waitForInvitePanel(page)
   return !(await page.isVisible(noPermissionToShareLabel))
 }
+
+export const copyQuickLink = async (args: copyLinkArgs): Promise<void> => {
+  const { page, resource, via } = args
+  if (via === 'CONTEXT_MENU') {
+    await clickActionInContextMenu({ page, resource }, 'create-quicklink')
+    await waitForPopupNotPresent(page)
```

Comment:
I don't  underatand here. what kind of popup are you not expecting?
when you press `create-quicklink` you create link. You can check POST response 
```
Request URL: https://localhost:9200/ocs/v1.php/apps/files_sharing/api/v1/shares
Request Method: POST
```

Reply:
The button is `Copy quicklink`, but its selector `oc-files-actions-create-quicklink-trigger` has `create-quicklink`. When the link is copied `The quicklink has been copied to your clipboard.` popup is shown and the function `waitForPopupNotPresent` waits for that popup to appear and disappear. Additionally, we can also check POST response.